canonical.json

```diff
@@ -220,6 +225,16 @@
             "Caisse d'épargne"
         ]
     },
+    "Caixa Geral de Depósitos":{
+        "matches":[
+            "Caixa Geral de Depósitos"
```

Comment:
If I'm seeing it well, you have aa match equal to the canonical name here. Did you mean something without accents or something like that?